lib/Twig/Error.php

```diff
@@ -135,6 +148,7 @@ public function getTemplateName()
     public function setTemplateName($name)
```

Comment:
There are some place in the code base that only have the template name locally, and rely on the guessing mechanism for filling the other info. Thus, I think we should keep this method asis: it would make little sense for these code paths to create a mock Twig_Source instance just to satisfy the new signature.